monkestation/code/modules/antagonists/borers/readme.md

```diff
@@ -0,0 +1,35 @@
+https://github.com/Monkestation/Monkestation2.0/pull/976
+
+## Cortical Borers
+
+MODULE ID: CORTICAL_BORERS
+
+### Description:
+
+This file is responsible for all cortical worm related files
+Cortical worms are antagonists whose sole purpose is to reproduce infintelly
+
+### TG Proc/File Changes:
+
+code\__DEFINES\role_preferences.dm
+code\__DEFINES\research\anomalies.dm
+code\_globalvars\lists\poll_ignore.dm
+code\datums\mutations\_mutations.dm
+code\modules\admin\sql_ban_system.dm
+code\modules\antagonists\changeling\powers\panacea.dm
+
+### Included files that are not contained in this module:
+
+code\__DEFINES\~monkestation\actionspeed_modification.dm
+code\__DEFINES\~monkestation\antagonists.dm
+code\__DEFINES\~monkestation\is_helpers.dm
+code\__DEFINES\~monkestation\role_preferences.dm
+
+tgui\packages\tgui\interfaces\BorerChem.jsx
+tgui\packages\tgui\interfaces\BorerEvolution.tsx
+
+### Credits:
+
+Jake Park - Original borer Code
```

Comment:
please credit me - I made both TGUI menus and reworked borers into having a skill tree and etc.